### ISSUE TITLE
Decouple status messages from actionable buttons.

### ISSUE THREAD

**Issue by jasonwebb** [jasonwebb/supertunnel#8]
Several buttons (like "Get my location") have their visible text updated dynamically to display status messages like "loading" or "locating". This is very confusing for screen reader users, since accessible names for buttons are expected to be static and represent the action that will happen when activated. This is also likely a violation of [WCAG 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).

Instead of changing the accessible name of the button, we should break these status messages out into separate text elements and show/hide them conditionally based on the app state. Per [WCAG 4.1.3](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html), making these messages available to screen reader users with an ARIA live region would be a good idea. At a high level, we should:

1. Remove the status messages from inside the button content, placing them as overlays on top to preserve the visual design.
2. Use an ARIA live region to announce the state changes to screen reader users.
3. When one of these non-actionable states is active, the button should be made inoperable, probably using the `disabled` attribute.

This issue needs to be done on the following buttons:

- [x] "Get my position" on desktop Origin panel.
- [x] "Get my location" on mobile step 1.
